generate-stackbrew-library.sh

```diff
@@ -71,7 +71,7 @@ for version in "${versions[@]}"; do
 	done
 	versionAliases+=( $version ${aliases[$version]:-} )
 
-	for variant in '' slim alpine; do
+	for variant in '' slim alpine jessie; do
 		dir="$version${variant:+/$variant}"
 		[ -f "$dir/Dockerfile" ] || continue
 
```

Comment:
yeah; we were upgrading from `jessie` to `stretch` but under some request (from another issue of this board) we added a variant of `jessie` was merging this PR but forgot the fact `jessie` has less number of architecture support, so should be more work than just this line change, the #165 looks good as a fix